sdk/cmake/host-tools.cmake

```diff
@@ -31,15 +31,30 @@ function(setup_host_tools)
     # custom target + symbolic output prevents cmake from running
     # the command multiple times per build
     add_custom_command(
-        COMMAND ${CMAKE_COMMAND} --build ${REACTOS_BINARY_DIR}/host-tools
+        COMMAND ${CMAKE_COMMAND} --build ${HOST_TOOLS_DIR}
         OUTPUT host_tools)
     add_custom_target(build-host-tools ALL DEPENDS host_tools)
 
-    include(${REACTOS_BINARY_DIR}/host-tools/ImportExecutables.cmake)
-    include(${REACTOS_BINARY_DIR}/host-tools/TargetList.cmake)
+    include(${HOST_TOOLS_DIR}/ImportExecutables.cmake)
+    include(${HOST_TOOLS_DIR}/TargetList.cmake)
 
     foreach(_target ${NATIVE_TARGETS})
         add_dependencies(native-${_target} build-host-tools)
     endforeach()
 
 endfunction()
+
+function(setup_host_tools)
+    if(WITH_HOST_TOOLS)
```

Comment:
I do not see any code set this variable. Does this mean that the ARM host tools needs to be built manually first?

Reply:
On windows with msvc yes, which is done by running cmake directly and without the TOOLCHAIN_FILE argument
with gcc the host tools should build automatically, its why i unset CC and CXX here
https://github.com/reactos/reactos/blob/9370741ce70034f0a0647983d4ec98522ea2fc8e/sdk/cmake/host-tools.cmake#L6-L12

Reply:
Then some comments in configure.cmd to explain this process might be helpful since this is an extra step.